index.js

```diff
@@ -32,9 +33,7 @@ module.exports = {
   },
 
   shouldIncludeFiles: function() {
-    var config = this.app.project.config()['ember-cli-mirage'];
-
-    return config.force || (this.app.env !== 'production')
+    return this.addonConfig.force || !this.addonConfig.usingProxy ? this.addonConfig.isEnabledEnvironment : true;
```

Comment:
Ah, I see. I can check and make sure it's working for both. The main reason I changed it was because it was being called multiple times without the environment being passed in, so it had environment as `undefined` for some of the times it was called.